### PR TITLE
Bug29858243 login error with db19 c

### PR DESCRIPTION
**Pull request by lananda**
Bug 548814 - EL not Auto Detecting Oracle 19C 

Added support for Oracle19C

### CODE REVIEW

File: foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/platform/database/Oracle19Platform.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ */
+
+// Contributors:
+//     IBM - initial drop
+//     09/19/2019 - Lalitha Ananda
+//       - 548814 : EL not Auto Detecting Oracle 19C.
+
+package org.eclipse.persistence.platform.database;
+
+public class Oracle19Platform extends Oracle12Platform {
```
Comment (rfelcman) [inline foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/platform/database/Oracle19Platform.java]: I think there shoud be `extends Oracle18Platform`. This is something like cascade.

File: foundation/org.eclipse.persistence.oracle/src/org/eclipse/persistence/platform/database/oracle/Oracle19Platform.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ */
+
+// Contributors:
+//     IBM - initial drop
+//     09/19/2019 - Lalitha Ananda
+//       - 548814 : EL not Auto Detecting Oracle 19C
+
+package org.eclipse.persistence.platform.database.oracle;
+
+public class Oracle19Platform extends Oracle12Platform {
```
Comment (rfelcman) [inline foundation/org.eclipse.persistence.oracle/src/org/eclipse/persistence/platform/database/oracle/Oracle19Platform.java]: I think there shoud be `extends Oracle18Platform`. This is something like cascade.

**Comment by lukasj** [comment]
oops, I forgot v18 has been already added, thanks @rfelcman !

File: foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/platform/database/Oracle19Platform.java (hunk 1 above)
Comment (lananda) [reply foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/platform/database/Oracle19Platform.java]: Thanks Radek. I've corrected the code & updated the pull request.

**Pull request merged**